plugins/synaptics-rmi/fu-synaptics-rmi-common.c

```diff
@@ -15,10 +15,8 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 
-#ifdef HAVE_GNUTLS
 #include <gnutls/abstract.h>
 #include <gnutls/crypto.h>
-#endif
```

Comment:
this was for the fuzzing; fuzzing with gnutls took a really long time.

Reply:
OK - will revert that